lib/msal-browser/docs/v4-migration.md

```diff
@@ -314,6 +315,94 @@ const authRequest = {
+
+- **Popup and silent flows**: The redirect bridge broadcasts the authentication response to the main application window using the BroadcastChannel API
+- **Redirect flow**: The redirect bridge navigates back to your application's home page with the authentication response in the URL
+
+#### How It Works
+
+1. **Main application**: Your COOP-enabled application initiates authentication using `loginPopup()`, `ssoSilent()`, or `loginRedirect()`
+2. **Redirect**: MSAL opens a popup/iframe/window to an authority page
+3. **Authentication flow**: The authority page completes the OAuth flow and receives the auth response
+4. **Response handling**: The redirect page uses the new `broadcastResponseToMainFrame()` function which:
+    - For **popup/silent flows**: Broadcasts the response to the main window via BroadcastChannel API
+    - For **redirect flows**: Navigates to your application's home page with the auth response
+5. **Token acquisition**: The main application receives the response and completes token acquisition
+
+#### Migration Steps
+
+##### 1. Set up your redirect URI page
+
+Create a separate HTML page (e.g., `redirect.html`) that will serve as your redirect bridge. **This page must NOT include COOP headers.**
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Redirect</title>
+</head>
+<body>
+    <script type="module">
+        import { broadcastResponseToMainFrame } from "@azure/msal-browser/redirect-bridge";
+
+        // Broadcast the auth response to the main application window
+        broadcastResponseToMainFrame()
+            .catch((error) => {
+                console.error("Error broadcasting response:", error);
+            });
+    </script>
+</body>
+</html>
+```
+
+##### 2. Configure your web server
+
+Ensure your redirect URI page is served **without** COOP headers, while your main application pages have COOP enabled:
+
+```javascript
+// Example using Express.js
+app.get("/redirect", (req, res) => {
+    // DO NOT set COOP headers for redirect page
+    res.sendFile(__dirname + "/redirect.html");
+});
+
+app.get("*", (req, res) => {
+    // Set COOP headers for all other pages
+    res.setHeader("Cross-Origin-Opener-Policy", "same-origin");
+    res.sendFile(__dirname + "/index.html");
+});
+```
+
+##### 3. Update your MSAL configuration
+
+Set the `redirectUri` in your MSAL configuration to point to your redirect bridge page:
+
+```javascript
+const msalConfig = {
+    auth: {
+        clientId: "{your-client-id}",
+        authority: "https://login.microsoftonline.com/common",
+        redirectUri: "https://{your-app-home-page}/redirect", // Point to your redirect bridge page
+    },
+};
+
+const pca = new PublicClientApplication(msalConfig);
+```
+
+- For a complete working example of COOP support with the redirect bridge, see the [COOP Sample Application](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/samples/msal-browser-samples/COOP).
```

Comment:
Let's take this out since the sample is temporary and for our own E2E use. I don't want to forget to remove the reference here.
```suggestion
```